saylua/modules/forums/models/db.py

```diff
@@ -26,6 +26,9 @@ class Board(db.Model):
 
     threads = db.relationship("ForumThread", back_populates="board")
 
+    def url(self):
```

Comment:
I'm glad to see this change, the previous ad-hoc method would lead to trouble down the line.